### ISSUE TITLE
inflator error (

### ISSUE THREAD

**Issue by ghost** [MariusVatasoiu/deno-image#3]
Sceenshot with error!
https://user-images.githubusercontent.com/75129392/100844940-d9a81000-34bf-11eb-8c38-651e5201e880.png

**Comment by MariusVatasoiu** [comment]
This will be fixed in the next release. 
Until then, you can use the `--no-check` flag as a quick fix.

**Comment by Nexxkinn** [comment]
@MariusVatasoiu Just for my feedback, you can use skydev's Pako build instead of manually bundling it into the module since It works perfectly fine on my modules, too.
```ts 
export { default as pako } from 'https://cdn.skypack.dev/pako?dts';
```

**Comment by MariusVatasoiu** [comment]
Fixed on v0.0.3.
